### PR TITLE
Adding ability to assign card to someone by their username, case insensitive

### PR DESCRIPTION
**Pull request by kedark3**
Added Card Command "assign" to assign card to someone as follows:

`(card RHCFQE-7382) assign bsquizza

(card RHCFQE-7382) assign kkulkarn

(card RHCFQE-7382) assign KKULKARN

(card RHCFQE-7382) assign BSQUIZZA

`

Also removed some whitespaces as lint was complaining.

### CODE REVIEW

File: simplejira/prompt.py [hunk 1]
```diff
@@ -222,6 +222,7 @@ def __init__(self, jira_wrapper, issue):
             'do_10': 'do_pull',
             'do_11': 'do_remove',
             'do_11': 'do_exit',
+            'do_12': 'do_assign',
```
Comment (bsquizz) [inline simplejira/prompt.py]: This is picky but I think exit should be last in the printed list of cmds -- so can we just move assign up to #11 and exit as #12

File: simplejira/prompt.py [hunk 2]
```diff
@@ -460,3 +461,10 @@ def do_editwork(self, args):
         collection = worklog_collection(worklogs)
         print(editor_ignore_comments(collection.to_yaml()))
     '''
+    assignee_parser = argparse.ArgumentParser()
+    assignee_parser.add_argument('assignee', const=None, type=str,
+                                 help="Assign card to someone as assign <username>;"
+                                 "username is case insensitive")
+    @cmd2.with_argparser(assignee_parser)
+    def do_assign(self, args):
+        self._jira.assign_issue(self.issue, args.assignee)
```
Comment (bsquizz) [inline simplejira/prompt.py]: We should add a check here for assignee being 'None':
```python
if not args.assignee:
    args.assignee = self.input("Enter assignee user id:")
```

File: simplejira/prompt.py (hunk 1 above)
Comment (kedark3) [reply simplejira/prompt.py]: I was thinking of that, but thought let you say it so and then I'll update the sequence. Not picky.

File: simplejira/prompt.py [hunk 3]
```diff
@@ -460,3 +461,19 @@ def do_editwork(self, args):
...
+    assignee_parser.add_argument('assignee', const=None, type=str,
+                                 help="Assign card to someone as assign <username>;"
+                                 "username is case insensitive")
+    @cmd2.with_argparser(assignee_parser)
+    def do_assign(self, args):
+        if not args.assignee:
+            args.assignee = self.input("Enter assignee user id:")
+        continue_assignment = 'Y'
+        if not args.assignee:
+            continue_assignment = self.input("Leaving assignee blank would unassign the card,"
```
Comment (bsquizz) [inline simplejira/prompt.py]: ah so for this, you can use 'prompter.yesno("Question")' and it'll automatically put the 'Y/n' and return true/false for you. So you can just do

```python
continue_assignment = prompter.yesno("Leaving assignee blank would unassign the card. Continue?")
if continue_assignment:
    stuff
else:
    stuff

**Pull request merged**